pages/partials/questionFooter.ejs

```diff
@@ -45,8 +35,14 @@
         <span class="d-flex">
           <input type="hidden" name="__variant_id" value="<%= variant.id %>">
           <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
-          <% if (showNewVariantButton) { %><a href="<%= newVariantUrl %>" class="btn btn-primary disable-on-click ml-1">New variant</a><% } %>
-          <% if (showTryAgainButton) { %><a href="<%= tryAgainUrl %>" class="btn btn-primary disable-on-click ml-1">Try question again</a><% } %>
+          <% if (showNewVariantButton) { %><a href="<%= newVariantUrl %>" class="btn btn-primary disable-on-click ml-1">New variant</a>
+          <% } else if (showTryAgainButton) { %><a href="<%= tryAgainUrl %>" class="btn btn-primary disable-on-click ml-1">Try a new variant</a>
+          <% } else if (hasAttemptsOtherVariants) { %>
+          <small class="font-italic align-self-center">Additional attempts available with new variants</small>
+          <button type="button" class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<%= include('newVariantInfo') %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
```

Comment:
When `data-trigger="focus"` it's necessary to use `<a>` rather than `<button>`: https://getbootstrap.com/docs/4.5/components/popovers/#dismiss-on-next-click

If we don't do this then the popovers break on Safari (see #3610).